src/main/java/de/plushnikov/intellij/plugin/language/LombokConfigCompletionContributor.java

```diff
@@ -41,7 +41,7 @@ public LombokConfigCompletionContributor() {
 
     final Collection<String> flagUsageOptions = new HashSet<>(Arrays.asList(
       "lombok.accessors.flagUsage", "lombok.allArgsConstructor.flagUsage", "lombok.anyConstructor.flagUsage",
-      "lombok.builder.flagUsage", "lombok.cleanup.flagUsage", "lombok.data.flagUsage", "lombok.delegate.flagUsage",
+      "lombok.builder.flagUsage", "lombok.cleanup.flagUsage", "lombok.copyableAnnotations", "lombok.data.flagUsage", "lombok.delegate.flagUsage",
```

Comment:
Why did you add it here, in the middle of only flagUsage options? Why do you think this is a good idea?

Maybe you can add it to "final Collection<String> otherOptions"?  And use a reference to your created key in from ConfigKey. (ConfigKey.COPYABLE_ANNOTATIONS..getConfigKey()). I think this would make more sense:)